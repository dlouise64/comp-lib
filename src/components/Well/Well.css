.well {
  background: rgb(243, 243, 243);
  padding: 35px 30px;
}
