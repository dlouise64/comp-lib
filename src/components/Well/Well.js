import React from 'react';
// import PropTypes from 'prop-types';

import './Well.css';

class Well extends React.Component {
  render() {
    return (
      <div className="well">Well</div>
    );
  }
}

Well.propTypes = {
};

Well.defaultProps = {
}

export default Well;
