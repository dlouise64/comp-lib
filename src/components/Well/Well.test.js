import React from 'react';
import renderer from 'react-test-renderer';
import Well from './Well';

test('<Well /> renders correctly', () => {
  const tree = renderer.create(<Well />).toJSON();
  expect(tree).toMatchSnapshot();
});
