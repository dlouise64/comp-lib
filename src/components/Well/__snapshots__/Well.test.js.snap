// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Well /> renders correctly 1`] = `<div />`;
